docs/changehistory/NextVersion.md

```diff
@@ -215,10 +209,11 @@ The following APIs were re-exported from `@itwin/core-bentley` and have been rem
 
 As of iTwin.js 5.0, the following packages have been removed and are no longer available:
 
-| Removed                        | Replacement                                                                                                            |
-| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------- |
-| `@itwin/core-webpack-tools`    | We no longer recommend using [webpack](https://webpack.js.org/) and instead recommend using [Vite](https://vite.dev/). |
-| `@itwin/backend-webpack-tools` | We no longer recommend webpack-ing backends, which was previously recommended to shrink the size of backends.          |
+| Removed                        | Replacement                                                                                                                                                 |
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `@itwin/backend-webpack-tools` | We no longer recommend bundling backends via webpack, which was previously recommended to shrink the size of backends.                                      |
```

Comment:
Can we add our recommended way to do this now?

Reply:
We dont recommend bundling backends at all, tried to make that more explicit now.